src/node_config.cc

```diff
@@ -73,27 +73,8 @@ static void Initialize(Local<Object> target,
 
 #endif  // NODE_HAVE_I18N_SUPPORT
 
-  if (env->options()->preserve_symlinks)
-    READONLY_TRUE_PROPERTY(target, "preserveSymlinks");
-  if (env->options()->preserve_symlinks_main)
-    READONLY_TRUE_PROPERTY(target, "preserveSymlinksMain");
-
-  if (env->options()->experimental_modules) {
-    READONLY_TRUE_PROPERTY(target, "experimentalModules");
-    const std::string& userland_loader = env->options()->userland_loader;
-    if (!userland_loader.empty()) {
-      READONLY_STRING_PROPERTY(target, "userLoader",  userland_loader);
-    }
-  }
-
-  if (env->options()->experimental_vm_modules)
-    READONLY_TRUE_PROPERTY(target, "experimentalVMModules");
-
-  if (env->options()->experimental_repl_await)
-    READONLY_TRUE_PROPERTY(target, "experimentalREPLAwait");
-
-  if (env->options()->expose_internals)
-    READONLY_TRUE_PROPERTY(target, "exposeInternals");
+  if (env->abort_on_uncaught_exception())
+    READONLY_TRUE_PROPERTY(target, "shouldAbortOnUncaughtException");
```

Comment:
This was removed in https://github.com/nodejs/node/commit/b06f2fafe7b6d61284cbaa03593bb645113fbd39 and should be kept out

Reply:
That's what the #25862 did. Should I remove this in this PR ?

Reply:
Yes, here you are re-introducing the line and it won't be removed again because the other PR is already backported. That's why we should really try not to backport out of order.

Reply:
Suggestion landed. Thank you :)